tests/testthat/test-cache.R

```diff
@@ -135,3 +135,45 @@ test_that("cache key components", {
   key4 <- sass_hash(list(input, options4))
   expect_true(key3 != key4)
 })
+
+
+
+test_that("output_file() is cache and options aware", {
```

Comment:
Should call `local_temp_cache()` at the top of this code block. This sets up a temporary cache and makes it the default.